brave/src/main/java/brave/Tracing.java

```diff
@@ -350,6 +354,32 @@ public Builder addFinishedSpanHandler(FinishedSpanHandler finishedSpanHandler) {
       return this;
     }
 
+    /**
+     * When true, all spans {@link TraceContext#sampledLocal() sampled locally} are reported to the
+     * {@link #spanReporter(Reporter) span reporter}, even if they aren't sampled remotely. Defaults
+     * to false.
+     *
+     * <p>The primary use case is to implement a sampling overlay, such as boosting the sample rate
+     * for a subset of the network depending on the value of an {@link ExtraFieldPropagation extra
+     * field}. This means that data will report when either the trace is normally sampled, or
+     * secondarily sampled via a custom header.
+     *
+     * <p>This is simpler than {@link #addFinishedSpanHandler(FinishedSpanHandler)}, because you
+     * don't have to duplicate transport mechanics already implemented in the {@link
+     * #spanReporter(Reporter) span reporter}. However, this assumes your backend can properly
+     * process the partial traces implied when using conditional sampling. For example, if your
+     * sampling condition is not consistent on a call tree, the resulting data could appear broken.
+     *
+     * @see #addFinishedSpanHandler(FinishedSpanHandler)
+     * @see TraceContext#sampledLocal()
+     * @see ExtraFieldPropagation.FactoryBuilder#addPlugin(ExtraFieldPlugin)
+     * @since 5.7
+     */
+    public Builder alwaysReportSpans() {
```

Comment:
ps to implement this in brave 5.7 we can use a finished span handler that wraps normal reporter and then sets tracing.builder.spanReporter to noop